internal/sirius/unlink_person.go

```diff
@@ -0,0 +1,37 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func (c *Client) UnlinkPerson(ctx Context, parentId int, childId int) error {
+	data, err := json.Marshal(struct {
+		ChildIds []int `json:"childIds"`
+	}{
+		ChildIds: []int{childId},
+	})
+
+	if err != nil {
+		return err
+	}
+
+	req, err := c.newRequest(ctx, http.MethodDelete, fmt.Sprintf("/api/v1/person-links/%d", parentId), bytes.NewReader(data))
+	if err != nil {
+		return err
+	}
+
+	resp, err := c.http.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
```

Comment:
this can return a standard `{"validation_errors" ...}` body, so should probably check and unmarshal it like we do elsewhere